Makefile

```diff
@@ -108,6 +109,30 @@ tag: ## Set a docker tag to the image.
 push: ## Push docker image.
 	docker push $(IMAGE_PREFIX)pvc-autoresizer:$(IMAGE_TAG)
 
+##@ Chart Testing
+
+.PHONY: ct-lint
+ct-lint: # Lint and validate a chart.
+	docker run \
+		--rm \
+		--user $(shell id -u $(USER)) \
+		--workdir=/data \
+		--volume $(shell pwd):/data \
+		quay.io/helmpack/chart-testing:v$(CHART_TESTING_VERSION) \
+		ct lint --config ct.yaml
+
+.PHONY: ct-install
+ct-install: #  Install and test a chart.
+	docker run \
+		--rm \
+		--user $(shell id -u $(USER)) \
+		--network host \
+		--workdir=/data \
+		--volume ~/.kube/config:/root/.kube/config:ro \
```

Comment:
By specifying user, it is unable to access to the directory due to permission.
Moreover, kubectl does not use the path, so ct install will fail.

Reply:
@toshipp Sorry. I fixed the `ct-lint` command. Would you review this PR again?